index.html

```diff
@@ -1,22 +1,152 @@
 <!DOCTYPE html>
-<html lang="en">
+<lang lang="en">
   <head>
     <meta charset="UTF-8" />
+    <meta name="description" content="Advise on keeping yourself secure on the internet">
+    <meta name="keywords" content="HTML, CSS, Cyber Security">
+    <meta name="author" content="Erin">
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-    <title>My Blog</title>
+    <title>Cyber Security in the 21st Century</title>
     <link
       href="//fonts.googleapis.com/css?family=Roboto:400,500,300"
       rel="stylesheet"
       type="text/css"
     />
     <link rel="stylesheet" href="css/style.css" />
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=KoHo:ital,wght@0,400;0,700;1,700&display=swap" rel="stylesheet">
   </head>
+
+
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    
+    <!-- the image and the h1 elements need to be inline so that display properly as I want them all on one line -->
+    
+    <header><div id="wrapper">
+
+      <div><image class="h1-image-1"
+        src="https://1gew6o3qn6vx9kp3s42ge0y1-wpengine.netdna-ssl.com/wp-content/uploads/prod/sites/5/2020/12/cyver.jpg"
+        alt="Locked paddlocks with a blue background"></image></div> 
+        <div>
+          <h1> Cyber Security in the 21st Century</h1>
+        </div>
+        <div>
+          <image class="h1-image-2"
+            src="https://1gew6o3qn6vx9kp3s42ge0y1-wpengine.netdna-ssl.com/wp-content/uploads/prod/sites/5/2020/12/cyver.jpg"
+            alt="Locked paddlocks with a blue background"></image>
+        </div>
+      
+      
+    
+      
+        
+    </div></header>
```

Comment:
There are a couple of improvements you can make in terms of code formatting here - for example putting child elements on a new line, removing some of the extra whitespace etc.

Also - the best tag to use for images is <img /> (it is self closing). <image /> will work in most browsers but is not an official part of the HTML tag set! which can be confusing